src/app/notifications/notification-type.ts

```diff
@@ -0,0 +1,9 @@
+export class NotificationType {
```

Comment:
We have notification types available in the Notification object of ngx-widgets. Although, we could create a separate NotificationType object if that's desirable?